airflow/jobs/__init__.py

```diff
@@ -16,8 +16,3 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-import airflow.jobs.backfill_job
```

Comment:
It is a bit risky. See: https://github.com/apache/airflow/pull/19185

Reply:
Found a good solution to this that I think is foolproof https://github.com/apache/airflow/pull/24486/commits/685498fa0800c3ca0e51ac41cbd3e863bdab4da6